### PR TITLE
Fix: consumer guide link (documentation)

### PR DESCRIPTION
**Pull request by nlamirault**

### CODE REVIEW

File: site/content/en/guides/producer/subpackages/_index.md [hunk 1]
```diff
@@ -214,4 +214,4 @@ Go through the [consumer guide] to consume the published package
 [auto-setter]: https://googlecontainertools.github.io/kpt/guides/producer/setters/#auto-setters
 [cfg]: https://googlecontainertools.github.io/kpt/reference/cfg/
 [subpackage]: https://googlecontainertools.github.io/kpt/concepts/packaging/#subpackages
-[consumer guide]: https://googlecontainertools.github.io/kpt/guides/producer/subpackages/
+[consumer guide]: https://googlecontainertools.github.io/kpt/guides/consumer/
```
Comment (frankfarzan) [inline site/content/en/guides/producer/subpackages/_index.md:217]: This should be
https://googlecontainertools.github.io/kpt/guides/consumer/subpackages/

**Review by prachirp — APPROVED** [review]
LGTM! @frankfarzan PTAL

**Comment by mikebz** [comment]
@frankfarzan bump 😄

**Review by frankfarzan — APPROVED** [review]

**Comment by mikebz** [comment]
@frankfarzan @mortent @droot should we merge this in?  Close it?

**Pull request merged**